xds/internal/balancer/edsbalancer/eds.go

```diff
@@ -196,6 +196,7 @@ func (x *edsBalancer) handleGRPCUpdate(update interface{}) {
 
 		x.config = cfg
 	case error:
+		x.logger.Warningf("Received error from resolver: %v", u)
 		x.handleErrorFromUpdate(u, true)
 	default:
 		// unreachable path
```

Comment:
The panic below makes me extremely nervous.

`log.Error` instead -- then tests will fail but production will just log an error.

Reply:
Done.